proxy/src/main/java/com/velocitypowered/proxy/network/TransportType.java

```diff
@@ -25,6 +25,8 @@
 import io.netty.channel.epoll.EpollEventLoopGroup;
 import io.netty.channel.epoll.EpollServerSocketChannel;
 import io.netty.channel.epoll.EpollSocketChannel;
+import io.netty.channel.epoll.EpollDomainSocketChannel;
```

Comment:
epoll is epoll, we can't just slap it on every single connection type, that is not how that works

Reply:
so i patch the paper

Reply:
so i patch the paper